pages/faq.rst

```diff
@@ -45,7 +45,7 @@ How do I become an instructor?
 
 First lets clear up a couple of misconcptions.  There is no way for me to know, just by your email address that you are an instructor.  You could be anyone.  So, there is no such thing as an instructor account.
 
-You become an instructor by creating a course. This gives you the privileges associated with being an instructor but ONLY for that course and any other course(s) you create.  You are not an instructor for any of the open courses or anyone else's course.  There is a really nice video to help you get started in the `Instructors Guide <https://runestone.academy/ns/books/published/instructorguide/frontmatter-1.html?mode=browsing>`_  Please check it out as it will answer many other questions and help you make the most of Runestone.
+You become an instructor by creating a course. This gives you the privileges associated with being an instructor but ONLY for that course and any other course(s) you create.  You are not an instructor for any of the open courses or anyone else's course.  There is a really nice video to help you get started in the `Instructors Guide <https://guide.runestone.academy/frontmatter-1.html>`.  Please check it out as it will answer many other questions and help you make the most of Runestone.
```

Comment:
The trailing `_` is required.  Please make sure you build and test your PRs.

Reply:
I didn't see any information on how to build the website. I thought submitting the fixed URL would be helpful but see now I was wrong. I apologize for wasting your time and will not submit any more Runestone PRs.

Thanks for maintaing Runestone.

Reply:
I don't mean to discourage you from submitting PRs.  

But you went beyond changing the URL and broke something.   I responded a bit more bluntly than necessary, due to my lack of sleep.  I don't think I even realized which repo this was against.  All you would need to do to your original PR is ad the `_` back after the final backquote.

I'll update the README so it has documentation for how to build.